docs/level-23.md

```diff
@@ -48,9 +49,16 @@ import TrashPush from '@site/image-generator/yml/level-23/trash-push.yml';
 
 <br />
 
-### Unnecessary Moves with Unknown Trash (Off-Chop) --> Chop Move
+### Unnecessary Moves with Unknown Trash
+- If Bob already has to play to tell the clue receiver that the clue target is trash, as an "extra thing" the team also interprets the clue as the **known trash** version of the clue.
+	- Applicable conventions here are the *Trash Finesse*, *Trash Bluff*, *Unknown Trash Discharge (UTD)*, *Bad Chop Move Ejection (BCME)*, and *Trash Push Discharge*.
```

Comment:
same here

Reply:
I thought it added clarity, because maybe it's not obvious when first learning whether BCME and trash push discharge are in the known trash set (since the 'focus' of the actual clue is trash) or unknown trash (since Bob needs to blind play to show the team the card next to the known trash is also trash).

Reply:
i think if you are level 23 it should be obvious, otherwise you are not actually level 23 and you need to play more games at lower levels

Reply:
one example of each thing is shown in the subsequent example in each section, that is probably good enough

Reply:
I meant when BCME is used unnecessarily, there's no example of that at all.
All 2s played.
Cathy has X - Y - r2 - g1
Alice clues 2 to Cathy, Bob ejects a card that could have been gotten directly. The categorization of 'known trash' or 'unknown trash' determines whether Cathy plays X or Y.

Reply:
Anyway, if you still think it's superfluous I'll delete the lines.

Reply:
then add an example

Reply:
okay, I'll rewrite one of the examples